subsys/mcuboot_ids/Kconfig

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menu "MCUboot IDs (informative only, do not change)"
+
+config MCUBOOT_APPLICATION_IMAGE_NUMBER
+	int "Application image number (informative only, do not change)"
```

Comment:
I know sysbuild defined setting values will always take precedence over image defined values.

But is there no way we can avoid prompt settings purely for the purpose of sharing this info ?

side-note, `do not change` is strictly not required, cause if a user does change the value, then the value from sysbuild will overrule.

Reply:
For sysbuild they will be forced, the warning however is for child/parent because they can freely be changed with it (which could break things). I think you submitted 2 different PRs to allow options to be set without a prompt from sysbuild -> images upstream but don't think either ever got finished